go/mysql/fastparse/fastparse.go

```diff
@@ -123,6 +123,9 @@ func ParseInt64(s string, base int) (int64, error) {
 		i++
 	}
 
+	if i >= uint(len(s)) {
```

Comment:
Added this check to avoid runtime error of `index out of range`.

Previously calling `ParseInt64("   ", 10)` resulted in runtime error `runtime error: index out of range [2] with length 2`.

Reply:
Good catch!